src/Dispatcher/GroupPosBased.php

```diff
@@ -13,6 +13,9 @@ protected function dispatchVariableRoute($routeData, $uri) {
                 continue;
             }
 
+            // to resolve vimeo/psalm PossiblyUndefinedVariable
+            $i = 1;
```

Comment:
forced-pushes & borked rebases are confusing github; what was dfe0d07e is now ec97c164b